stream/distributedlog/protocol/src/main/java/org/apache/distributedlog/exceptions/UnsupportedMetadataVersionException.java

```diff
@@ -25,6 +25,6 @@
     private static final long serialVersionUID = 4980892659955478446L;
 
     public UnsupportedMetadataVersionException(String message) {
-        super(StatusCode.UNSUPPORTED_METADATA_VERSION, String.format(message));
+        super(StatusCode.UNSUPPORTED_METADATA_VERSION, message);
```

Comment:
I think this is a behavior change

Reply:
thanks for the comment. The change was reverted, but I don't see how this formatting can help without any arguments

Reply:
You are right, but it is better not to change things if not needed or out of the scope of the patch.
Sorry for so deep nitpicking :)